setup.py

```diff
@@ -31,6 +31,8 @@
 
 install_requires = [
     'BeautifulSoup>=3.2.1',
+    # sentry also requires this, so we're just enforcing that it needs to exist
+    'boto3>=1.4.4,<1.5.0',
```

Comment:
Just heads up, I don't see why you'll require 1.4.4. We have it pinned to 1.4.1 in getsentry today, so we can either loosen this up here, or bump it in getsentry or we'll be conflicting.